umbra-js/src/utils/utils.ts

```diff
@@ -241,6 +242,17 @@ export async function lookupRecipient(
   return { spendingPublicKey: publicKey, viewingPublicKey: publicKey };
 }
 
+export async function getBlockNumberUserRegistered(address: string, startblock = 0, Signer: JsonRpcSigner) {
+  address = getAddress(address); // address input validation
+  const { chainId } = await Signer.provider.getNetwork();
+  const txHistoryProvider = new TxHistoryProvider(chainId);
+  const history = await txHistoryProvider.getHistory(address, startblock);
+  // Address of the StealthKeyRegistry is the same on all supported networks
+  const StealthKeyRegistryAddress = '0x31fe56609C65Cd0C510E7125f051D440424D38f3';
```

Comment:
It might be good to move this reference and the reference in `StealthKeyRegistry.ts` to a constants file.

Reply:
Moved it to constants here: https://github.com/ScopeLift/umbra-protocol/pull/549/commits/b9b27f8a7559f6434be4a9356a5578e2f9509861